.dockerignore

```diff
@@ -1 +1,5 @@
 .env
+.env.example
+.git
+.github
+node_modules
```

Comment:
Great. We could also think about ignoring everything out of the box and then explicitly allowing the files/folders that we want to have in the Dockerfile. Wdyt?

Reply:
do you mean we copy required files and folders directly from Dockerfile rather than ignoring some of them?

Reply:
This would be the one option (also often recommended by tools like SonarCloud). The other option (and what I meant) is to just ignore everything in the docker ignore and allow them explicitly like so
```
*
!allowed_file.example
```
See e.g. [here](https://github.com/digitalservicebund/grundsteuer/blob/main/.dockerignore) for Grundsteuer

Reply:
ah thanks! I will have a look